content/millennium/r4/documents/document-reference.md

```diff
@@ -160,6 +160,62 @@ X-Request-Id: 3e4cb2f732daacdb6cca2eb944e80e55
 X-Runtime: 2.011826
 </pre>
 
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Update
+
+Update an existing document reference.
+
+    PUT /DocumentReference/:id
+
+_Implementation Notes_
+
+* Any field which is missing will be interpreted as nulling out or removing data from the resource. See [FHIR<sup>®</sup> Update] for additional details about update operations.
+* Both write and read scopes are required to update a document reference
```

Comment:
we should no longer need read scopes for an update, we can remove this line.

Reply:
You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/463/commits/2697a167ece46ac0a9e9b0a2a671208d4ce9715d